state/runtime/tracer/structtracer/tracer.go

```diff
@@ -0,0 +1,382 @@
+package structtracer
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"sync"
+
+	"github.com/0xPolygon/polygon-edge/helper/hex"
+	"github.com/0xPolygon/polygon-edge/state/runtime"
+	"github.com/0xPolygon/polygon-edge/state/runtime/evm"
+	"github.com/0xPolygon/polygon-edge/state/runtime/tracer"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+type Config struct {
+	EnableMemory     bool // enable memory capture
+	EnableStack      bool // disable stack capture
```

Comment:
Is this comment by purpose like that, since it is following a spec? I find it misleading though, because `EnableStack` and `EnableStorage` are set as negated values of `DisableStack` and `DisableStorage` set from the end-user.

```suggestion
	EnableStack      bool // enable stack capture
```

Reply:
I forgot changing when I created a config object in jsonrpc layer. The interface is following the spec but this Config object doesn't have to follow it.

Reply:
569552c745e4054b86bd3966bb2a1bdac1f048f3